udata/core/organization/api_fields.py

```diff
@@ -43,7 +48,18 @@
     'user': fields.Nested(user_ref_fields),
     'role': fields.String(
         description='The member role in the organization', required=True,
-        enum=list(ORG_ROLES), default=DEFAULT_ROLE)
+        enum=list(ORG_ROLES), default=DEFAULT_ROLE),
+    'since': fields.ISODateTime(
+        description='The date the user joined the organization', readonly=True),
+})
+
+private_member_fields = api.model('PrivateMember', {
+    'user': fields.Nested(user_in_org_with_email_fields),
+    'role': fields.String(
+        description='The member role in the organization', required=True,
+        enum=list(ORG_ROLES), default=DEFAULT_ROLE),
+    'since': fields.ISODateTime(
```

Comment:
I'm thinking `since` could be a public info available to all.

Reply:
It is :-) It's present in the public field above